packages/lambda/src/handlers/get-long-url.ts

```diff
@@ -26,9 +27,13 @@ export const getLongUrlHandler: Handler = async (event) => {
 
   console.log("Received short URL ID: ", shortUrlId);
 
-  const { Item } = await dynamoClient.send(new GetCommand({ TableName: URLS_TABLE_NAME, Key: { shortUrlId } }));
-  if (!Item) throw new NotFound(`Could not find a long URL from the shortUrlId: ${shortUrlId}`);
+  // Fetch the URL details from DynamoDB and publish the analytics to Firehose in parallel
+  const [{ Item }] = await Promise.all([
```

Comment:
To save time, we're publishing the analytics and reading from DynamoDB at the same time. I've just realised that this needs to be `allSettled` instead of `all` so that we don't fail if publishing the analytics fails. Will fix this now.

Reply:
(done)